modules/block-interactions/src/main/java/unnamed/mmo/blocks/ore/handler/OreBlockHandler.java

```diff
@@ -81,11 +83,15 @@ public void onDestroy(@NotNull Destroy destroy) {
             return;
         }
 
+        // Call break event
+        final Player player = ((PlayerDestroy) destroy).getPlayer();
```

Comment:
Could potentially cause a classcastexception in the far off future, may want to do a pattern-matched cast here

Reply:
It was being checked above, but a bit hidden. I made it more clear now.